### DISCUSSION TITLE
PhotoPrism does not start after update

### DISCUSSION THREAD

**Discussion by DimonXe** [tteck/Proxmox#3057] [Help Needed] [answered]
Hi all,
PhotoPrism does not start after update.
The service will not start manually either.
I'm attaching an error message. 

`Job for photoprism.service failed because the control process exited with error code.
See "systemctl status photoprism.service" and "journalctl -xeu photoprism.service" for details.

[ERROR] in line 71: exit code 0: while executing command sudo systemctl start photoprism

root@photoprism:# systemctl start photoprism
Job for photoprism.service failed because the control process exited with error code.
See "systemctl status photoprism.service" and "journalctl -xeu photoprism.service" for details.
root@photoprism:~# systemctl status photoprism.service
x photoprism.service - PhotoPrism service
     Loaded: loaded (/etc/systemd/system/photoprism.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Fri 2024-05-24 17:28:45 CEST; 12s ago
    Process: 365 ExecStart=/opt/photoprism/bin/photoprism up -d (code=exited, status=127)
        CPU: 941us

May 24 17:28:45 photoprism systemd[1]: Starting PhotoPrism service...
May 24 17:28:45 photoprism photoprism[365]: /opt/photoprism/bin/photoprism: error while loading shared libraries: libvips.so.42: cannot open shared object file: No such file or directory
May 24 17:28:45 photoprism systemd[1]: photoprism.service: Control process exited, code=exited, status=127/n/a
May 24 17:28:45 photoprism systemd[1]: photoprism.service: Failed with result 'exit-code'.
May 24 17:28:45 photoprism systemd[1]: Failed to start PhotoPrism service.`

**Comment by tteck** [comment, marked as answer]
Try installing `apt-get install -y libvips42`

**Reply by DimonXe** [reply]
> Try installing `apt-get install -y libvips42`

It's working. Thanks for the quick response!
